src/config.cpp

```diff
@@ -204,9 +204,9 @@ static std::expected<int64_t, std::string> configStringToInt(const std::string&
             if (!r.has_value() || !g.has_value() || !b.has_value())
                 return std::unexpected("failed parsing " + VALUEWITHOUTFUNC);
 
-            return a * 0x1000000L + r.value() * 0x10000L + g.value() * 0x100L + b.value();
+            return a * INT64_C(0x1000000) + r.value() * INT64_C(0x10000) + g.value() * INT64_C(0x100) + b.value();
```

Comment:
wut da foq is `INT64_C`? don't we have `int64_t`?

Reply:
I suppose a cast would be better than the macro...

Reply:
Switched them all to cast to Hyprlang::INT (which is int64_t), still works.